core/src/bson_type_info.rs

```diff
@@ -218,6 +218,52 @@ impl BsonTypeInfo {
         column_size: make_default_attr_func!(Some(MAX_STRING_SIZE)),
         simple_type_info: None,
     };
+    pub const WLONGVARCHAR: BsonTypeInfo = BsonTypeInfo {
```

Comment:
Can you add a similar comment than the one on top of "VARCHAR" explaining that this was added for PowerBI and is necessary because of the way the apply a cast using the type name.